scripts/experiments_tabular/knn_run.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 #SBATCH --time=24:00:00
-#SBATCH --nodes=1 --ntasks-per-node=1 --cpus-per-task=1
+#SBATCH --nodes=1 --ntasks-per-node=2 --cpus-per-task=1
```

Comment:
Have you checked if it helps with the number of submission we can make to the slurm scheduler/queue?

Reply:
Not yet, I think this should just speed things a little bit.